packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -131,19 +115,29 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
   }, [sortBy]);
 
   return isLoading ? (
-    <Box className={classes.loadingIndicatorContainer}>
-      <CircularProgress className={classes.loadingIndicator} />
+    <Box
```

Comment:
trying to work with what this says https://next.material-ui.com/system/basics/

This is basically inline styles.

... I kinda.. like it....? I think?